src/core/tests/pass/serialization/serialize.cpp

```diff
@@ -40,15 +42,17 @@ class SerializationTest : public ov::test::TestsCommon, public testing::WithPara
 };
 
 TEST_P(SerializationTest, CompareFunctions) {
+    auto orig = ov::test::readModel(m_model_path, m_binary_path);
     auto expected = ov::test::readModel(m_model_path, m_binary_path);
```

Comment:
Looks like you read the same model two times.
You can just clone_model

Reply:
done